I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00library.rpy", line 28, in script call
    call _errorhandling
AttributeError: 'StoreModule' object has no attribute '_default'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Utilisateur\Desktop\Awen\renpy-7.1.1-sdk\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "C:\Users\Utilisateur\Desktop\Awen\renpy-7.1.1-sdk\renpy\main.py", line 423, in main
    renpy.game.context().run(node)
  File "renpy/common/00library.rpy", line 28, in script call
    call _errorhandling
  File "renpy/common/_errorhandling.rpym", line 245, in script
    python hide:
  File "C:\Users\Utilisateur\Desktop\Awen\renpy-7.1.1-sdk\renpy\ast.py", line 882, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Utilisateur\Desktop\Awen\renpy-7.1.1-sdk\renpy\python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/_errorhandling.rpym", line 245, in <module>
    python hide:
  File "renpy/common/_errorhandling.rpym", line 263, in _execute_python_hide
    style._default.hyperlink_functions = (hyperlink_styler, hyperlink_function, None)
AttributeError: 'StoreModule' object has no attribute '_default'

Windows-8-6.2.9200
Ren'Py 7.1.1.929
 
Thu Oct 25 15:45:05 2018
